### PR TITLE
Add documentation for running OP replica

### PR DESCRIPTION
**Pull request by boyuan-chen**
This PR includes a doc for running erigon and op-node as the replica for Optimism.

### CODE REVIEW

**Review by jyellick — APPROVED** [review]
Not sure I love the filename of `OP.md`, and it might be worth simply extracting some of these embedded scripts into actual scripts, but, it's a good starting place and certainly good enough to get me going.

File: OP.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+```bash
+cd v3-erigon
+./build/bin/erigon \
+--datadir=DATA_DIRECTORY \
+--private.api.addr=localhost:9090 \
+--http.addr=0.0.0.0 \
+--http.port=9545 \
+--http.corsdomain="*" \
+--http.vhosts="*" \
+--authrpc.addr=0.0.0.0 \
```
Comment (mmontour1306) [inline OP.md]: You have trailing whitespace after the final backslash, here and on the disabletxpoolgossip line, which breaks the line continuation if you copy-paste from this file into a terminal.

File: OP.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+--datadir=DATA_DIRECTORY \
+--private.api.addr=localhost:9090 \
+--http.addr=0.0.0.0 \
+--http.port=9545 \
+--http.corsdomain="*" \
+--http.vhosts="*" \
+--authrpc.addr=0.0.0.0 \ 
+--authrpc.port=8551 \
+--authrpc.vhosts="*" \
+--authrpc.jwtsecret=JWT_TOKEN_DIRECTORY \
```
Comment (mmontour1306) [inline OP.md]: This should be called JWT_TOKEN_PATH as it needs the actual file, not its directory.

File: OP.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+```bash
+INFO [08-04|16:36:01.204] Found next batch                         epoch=44e203..fef9a5:8301309 batch_epoch=8301309                batch_timestamp=1,673,567,518
+INFO [08-04|16:36:01.205] generated attributes in payload queue    txs=2  timestamp=1,673,567,518
+INFO [08-04|16:36:01.265] inserted block                           hash=ee61ee..256300 number=4,069,725 state_root=a582ae..33a7c5 timestamp=1,673,567,518 parent=5b102e..13196c prev_randao=4758ca..11ff3a fee_recipient=0x4200000000000000000000000000000000000011 txs=2  update_safe=true
+```
+
+### Note
+
+It is best to run `boba-erigon` first and shut it down last.
```
Comment (mmontour1306) [inline OP.md]: More than "best to" - it's required to have the erigon process running first or else op-node will error out.

File: OP.md (hunk 1 above)
Comment (boyuan-chen) [reply OP.md]: Good catch! Fixed

File: OP.md (hunk 3 above)
Comment (boyuan-chen) [reply OP.md]: I change the word to `must`

**Review by mmontour1306 — APPROVED** [review]

**Pull request merged**